solr/contrib/ltr/example/config.json

```diff
@@ -4,11 +4,14 @@
   "collection":               "techproducts",
   "requestHandler":           "query",
   "q":                        "*:*",
-  "otherParams":              "fl=id,score,[features efi.user_query='$USERQUERY']",
+  "efiParams":                "efi.user_query='$USERQUERY'",
   "userQueriesFile":          "user_queries.txt",
-  "trainingFile":             "ClickData",
-  "featuresFile":             "exampleFeatures.json",
+  "trainingFile":             "exampleTrainingFile.txt",
+  "trainedModelFile":         "exampleTrainedModel.txt",
   "trainingLibraryLocation":  "liblinear/train",
+  "trainingLibraryOptions":   "-q",
+  "featuresFile":             "exampleFeatures.json",
+  "featureStoreName":         "exampleFeatureStore",
```

Comment:
should these be prefixed like the model versions?  solrFeaturesFile, solrFeatureStoreName?